### PR TITLE
react1 week1 homework

### PR DESCRIPTION
**Pull request by faheem102**

### CODE REVIEW

File: react/react1/week1/my-app/src/Listitems.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import React from "react";
+export function Listitems(props){
+    return (
```
Comment (senner007) [inline react/react1/week1/my-app/src/Listitems.js:3]: ListItem singular and not ListItem**s**, right?

File: react/react1/week1/my-app/src/TodoList.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import {Listitems} from "./Listitems";
+export function TodoList(props){
+    return (
```
Comment (senner007) [inline react/react1/week1/my-app/src/TodoList.js:4]: Again, I prefer not to inline the map operations in the JSX. Assign it to a const before the return and give it a meaningful name

File: react/react1/week1/my-app/src/Listitems.js (hunk 1 above)
Comment (faheem102) [reply react/react1/week1/my-app/src/Listitems.js]: hi thank you for the feedback. I have now made this as Listitem singular as suggested.

File: react/react1/week1/my-app/src/TodoList.js (hunk 2 above)
Comment (faheem102) [reply react/react1/week1/my-app/src/TodoList.js]: as per your suggestion I have made map operation outside the JSX and assigned it to a const. please review the changes and let me know if I can merge my pull request. thank you

**Pull request merged**